var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -297,6 +297,7 @@ class Gcc(AutotoolsPackage, GNUMirrorPackage):
         patch('https://raw.githubusercontent.com/Homebrew/formula-patches/22dec3fc/gcc/gcc-11.3.0-arm.diff',
               sha256='e02006b7ec917cc1390645d95735a6a866caed0dfe506d5bef742f7862cab218',
               when='@11.3.0 target=aarch64:')
+        conflicts('+bootstrap', when='@11.3.0 target=aarch64')
```

Comment:
```suggestion
        conflicts('+bootstrap', when='@11.3.0 target=aarch64:')
```
Open ended range needed to cover M1 and other specific architectures